### PR TITLE
Update tiup document

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)

Update tiup docs.
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4117
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: scale-tidb-using-tiup.md [hunk 1]
```diff
@@ -109,6 +109,10 @@ If you want to add a TiDB node to the `10.0.1.5` host, take the following steps.
     tiup cluster scale-out <cluster-name> scale-out.yaml
     ```
 
+    > **Note:**
+    >
+    > Assuming that the current executing command and the newly added machine establish mutual trust. If the conditions for establishing mutual trust are not met, you need to enter the password of the new machine through `-p`, or specify the private key file through `-i`.
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:114]:
```suggestion
    > The command above is based on the assumption that the mutual trust has been configured for the user to execute the command and the new machine. If the mutual trust cannot be configured, use the `-p` option to enter the password of the new machine, or use the `-i` option to specify the private key file.
```

File: scale-tidb-using-tiup.md [hunk 2]
```diff
@@ -163,6 +167,10 @@ If you want to add a TiFlash node to the `10.0.1.4` host, take the following ste
...
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:172]: ditto

File: scale-tidb-using-tiup.md [hunk 3]
```diff
@@ -207,6 +215,10 @@ If you want to add two TiCDC nodes to the `10.0.1.3` and `10.0.1.4` hosts, take
...
```
Comment (TomShawn) [inline scale-tidb-using-tiup.md:220]: ditto

File: tiup/tiup-faq.md [hunk 4]
```diff
@@ -10,6 +10,11 @@ aliases: ['/docs/dev/tiup/tiup-faq/']
 
 TiUP supports specifying the mirror source through the `TIUP_MIRRORS` environment variable. The address of the mirror source can be a local directory or an HTTP server address. If your environment cannot access the network, you can create your own offline mirror source to use TiUP.
 
+If you want to switch the unofficial mirror back to the official mirror, take any of the following measures:
```
Comment (TomShawn) [inline tiup/tiup-faq.md:13]:
```suggestion
After using an unofficial mirror, if you want the official mirror back and use it, take one of the following measures:
```

File: tiup/tiup-faq.md [hunk 5]
```diff
@@ -10,6 +10,11 @@ aliases: ['/docs/dev/tiup/tiup-faq/']
...
+
+- Set the `TIUP_MIRRORS` variable to the official mirror address: `https://tiup-mirrors.pingcap.com`.
+- Make sure that you do not set the `TIUP_MIRRORS` variable, and then execute the `tiup mirror set https://tiup-mirrors.pingcap.com` command.
```
Comment (TomShawn) [inline tiup/tiup-faq.md:16]:
```suggestion
- Make sure that the `TIUP_MIRRORS` variable is not set, and then execute the `tiup mirror set https://tiup-mirrors.pingcap.com` command.
```

**Review by TomShawn — APPROVED** [review]
LGTM
@lucklove

**Review by lucklove — APPROVED** [review]
LGTM

**Pull request merged**